src/scripts/initData.sjs

```diff
@@ -36,32 +40,36 @@ while (remainingCount > 0) {
     //body
     + "<SOAP-ENV:Body>"
     + "<ns1:paramsGetMultipleLeads xmlns:ns1=\"http://www.marketo.com/mktows/\">"
-    + "<lastUpdatedAt>2016-07-19T11:51:08.710-08:00</lastUpdatedAt>"
-    + "<batchSize>100</batchSize>"
+    + "<lastUpdatedAt>2016-02-02T11:51:08.710-08:00</lastUpdatedAt>"
```

Comment:
instead of hard-coding this here, declare a variable near the top and document how someone should decide the value. We'll probably need to use a different value the next time we do an EA.